ronn/geopm_agent_frequency_map.7.ronn

```diff
@@ -73,7 +73,9 @@ name (see **geopm(7)**).  This name can also be passed to the
                   specified with a CPU frequency that is allowed by
                   the system.  Failure to provide this parameter,
                   setting it to an out of range value or specifying
-                  NAN will result in a runtime error.
+                  NAN will result in a runtime error, unless the policy
+                  is all NAN, in which case the agent will not exert
+                  any control.
```

Comment:
Why allow an all NAN policy?  Why have this agent not exert control?  We have the monitor for this use case.

Reply:
This issue is around the case where a non-root controller starts up and has not yet received a policy from it's parent.  In this case, we don't want the agent to do anything until a policy with values arrives.